### ISSUE TITLE
Bug in building.calc_three_element

### ISSUE THREAD

**Issue by PRemmen** [RWTH-EBC/TEASER#294]
line 494-496
```python
if self.r_rad_inner_gf != 0:
    self.r_rad_inner_ow = 1/((1/self.r_rad_inner_ow)+(1/self.r_rad_inner_rt))
```

must be
```python
if self.r_rad_inner_rt != 0:
    self.r_rad_inner_ow = 1/((1/self.r_rad_inner_ow)+(1/self.r_rad_inner_rt))
```

**Comment by MichaMans** [comment]
Isn't this in the thermal_zone module and class?

**Comment by PRemmen** [comment]
Yes...

**Comment by MichaMans** [comment]
solved with #311
